doc/api/fs.markdown

```diff
@@ -989,6 +989,32 @@ The synchronous version of [`fs.writeFile()`][]. Returns `undefined`.
 
 Synchronous versions of [`fs.write()`][]. Returns the number of bytes written.
 
+## fs.mkdtemp(template, callback)
+
+Creates a unique temporary directory.
+
+The `fs.mkdtemp()` function generates a uniquely named temporary
+directory from template. The last six characters of template must be
+XXXXXX and these are replaced with a string that makes the directory
```

Comment:
> I disagree. mkdtemp is a well-known function across languages with a well-defined API. Changing this API in node.js feels very wrong.

This depends on the language and [python](https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp) for example does not use this notation.

> Can't we just say that the temporary directory will be template + six additional characters and have the internals of the function handle the rest?

+1